include/rangers-api/Application/UI/UIMusicSelect.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+namespace app::ui {
+    class UIMusicSelect : public hh::game::GameObject, public hh::ui::UIListener, public hh::hid::ActiveDeviceManagerListener, public app::snd::CustomSoundListener {
+        hh::ui::LayerController* layerController;
+        hh::ui::LayerController* refBtn;
+        hh::ui::LayerController* refPcBtnT;
+        hh::ui::LayerController* refPcBtnB;
+        float float1;
+        bool field_284;
+        hh::game::GOCInput* gocInput;
+        csl::ut::MoveArray<void*> songs;
+        csl::ut::MoveArray<void*> field_2B0;
+        uint32_t playingSong;
+        uint32_t playingSongIdx;
+        uint32_t songCount;
+        bool field_2DC;
+        bool inlooporloop;
```

Comment:
This variable makes no sense. Also keep the variable name as unknown unless it can be reasonably named and the functionality of it due to it is known. Ideas can be kept to comments.